### PR TITLE
Add Topics to Project

### PR DESCRIPTION
**Pull request by lwalen**
`tag_list` was deprecated in GitLab 14. https://docs.gitlab.com/ee/api/projects.html

### CODE REVIEW

File: projects.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ type Project struct {
 	HTTPURLToRepo                             string                     `json:"http_url_to_repo"`
 	WebURL                                    string                     `json:"web_url"`
 	ReadmeURL                                 string                     `json:"readme_url"`
+	Topics                                    []string                   `json:"topics"`
```
Comment (svanharmelen) [inline projects.go:51]: These fields are ordered using to ordering in the docs. For this field that means it should be placed between just below the `TagList` field.

**Review by svanharmelen — CHANGES_REQUESTED** [review]
One super minor change request, but looks fine otherwise 👍🏻

File: projects.go (hunk 1 above)
Comment (lwalen) [reply projects.go]: Done, thanks!

**Review by svanharmelen — APPROVED** [review]
Thanks @lwalen 👍🏻

**Comment by Deichindianer** [comment]
Is there any indication when this might be released? Currently waiting for this feature to write some systems around topics.

If there is something left to do, I'm also happy to try and help out :)

**Comment by svanharmelen** [comment]
Just released version `v0.50.4`

**Pull request merged**